aws_lambda_builders/workflows/java_gradle/gradle_validator.py

```diff
@@ -0,0 +1,60 @@
+"""
+Gradle Binary Validation
+"""
+
+import logging
+import re
+
+from aws_lambda_builders.exceptions import LambdaBuilderError
+from .utils import OSUtils
+
+LOG = logging.getLogger(__name__)
+
+
+class GradleBinaryValidatorError(LambdaBuilderError):
+    MESSAGE = "gradle executable found failed to return version"
```

Comment:
Can you make this more descriptive? This error message is opaque

Reply:
Will do!